.github/workflows/ci.yml

```diff
@@ -43,6 +43,9 @@ jobs:
       - name: Build Claude MCP Bundle
         run: npm run build:mcpb
 
+      - name: Build NPM package
+        run: npm pack
```

Comment:
This isn't doing anything other than verifying that we can _create_ an NPM package and CI will be blocked if there are any errors.